src/dotnet/commands/dotnet-build/README.md

```diff
@@ -1,38 +1,87 @@
-dotnet-build
-===========
+% DOTNET-BUILD(1)
+% Zlatko Knezevic zlakne@microsoft.com
```

Comment:
We actually did this because man pages usually have these. Remove the email, author or both?